google/cloud/storage/client.py

```diff
@@ -1480,6 +1530,57 @@ def list_buckets(
             retry=retry,
         )
 
+    def restore_bucket(
+        self,
+        bucket_name,
+        generation,
+        if_metageneration_match=None,
+        if_metageneration_not_match=None,
+        timeout=_DEFAULT_TIMEOUT,
+        retry=DEFAULT_RETRY,
```

Comment:
Let's check with the API folks... it sounds more like `DEFAULT_RETRY_IF_GENERATION_SPECIFIED`?